### PR TITLE
feat(apps/interface): add my swap container

### PR DESCRIPTION
**Pull request by asaadam**

### CODE REVIEW

**Comment by pyk** [comment]
@asaadam your implementation is different from the UI spec

Screen Shot 2022-08-16 at 11 34 08
Screen Shot 2022-08-16 at 11 35 06

**Review by pyk — CHANGES_REQUESTED** [review]
see my comment above

File: apps/interface/components/SwapHistoryCard/Table.tsx [hunk 1]
```diff
@@ -22,10 +24,64 @@ import ArrowTopRight from "@/components/Icons/ArrowTopRight";
...
+            <Text
+                color={gray12}
+                fontSize="sm"
+                lineHeight="4"
+                margin="0 !important"
+                data-testid="BackingCardCollateral"
+            >
+                {swap.tokenIn.symbol === currentCollateral
+                    ? `${transactionType} ${swap.tokenOut.symbol} `
+                    : `${transactionType} ${swap.tokenIn.symbol}`}
```
Comment (risedledev) [inline apps/interface/components/SwapHistoryCard/Table.tsx:71]: - buy

```
tokenOut = fltTokenSymbol
```

- sell

```
tokenIn = fltTokenSymbol
```

**Review by risedledev — CHANGES_REQUESTED** [review]

**Review by pyk — CHANGES_REQUESTED** [review]
The implementation is not same as designed by @ayikfour 

Screen Shot 2022-08-26 at 10 29 10

expected:
Screen Shot 2022-08-26 at 10 33 38

File: apps/interface/components/SwapHistoryCard/Table.tsx [hunk 2]
```diff
@@ -152,18 +178,21 @@ export const SwapHistoryCardTable = (props: SwapHistoryCardTableProps) => {
...
-                                color={gray12}
-                                fontSize="sm"
-                                lineHeight="4"
-                                margin="0 !important"
-                            >
-                                <TimeAgo
-                                    date={parseInt(swap.timestamp) * 1000}
-                                />
-                            </Text>
+                            <Flex minH={"48px"} alignItems={"center"}>
```
Comment (pyk) [inline apps/interface/components/SwapHistoryCard/Table.tsx:183]: @asaadam remove the `minH`

File: apps/interface/components/SwapHistoryCard/Table.tsx [hunk 3]
```diff
@@ -199,29 +228,31 @@ export const SwapHistoryCardTable = (props: SwapHistoryCardTableProps) => {
...
-                            <Text
-                                color={gray12}
-                                fontSize="sm"
-                                lineHeight="4"
-                                fontFamily="mono"
-                                letterSpacing="tight"
-                            >
-                                {formatUSD(parseFloat(swap.amountInUSD))}
-                            </Text>
+                            <Flex minH={"48px"} alignItems={"center"}>
```
Comment (pyk) [inline apps/interface/components/SwapHistoryCard/Table.tsx:244]: @asaadam remove the `minH`

**Review by pyk — CHANGES_REQUESTED** [review]
Smol nitpicks, otherwise LGTM. lets merge this

**Review by pyk — CHANGES_REQUESTED** [review]
Btw it seems that u miss another `minH`
Screen Shot 2022-08-29 at 20 55 45

**Review by risedledev — APPROVED** [review]
LGTM, kita deploy dulu yah @asaadam @math-marcellino

**Review by pyk — APPROVED** [review]

**Pull request merged**